src/frontend/src/utils/hasNonEmptyStringProperties.ts

```diff
@@ -0,0 +1,3 @@
+export const hasNonEmptyStringProperties = (inputObject: any) => {
```

Comment:
```suggestion
export const hasNoEmptyStringProperties = (inputObject: any) => {
```
hasNonEmptyStringProperties would mean return true if any property is not empty
hasNoEmptyStringProperties would mean return true if the object has no properties with an empty string

Reply:
@BradyMitch Changed it! :)